cfme/v2v/migration_plans.py

```diff
@@ -0,0 +1,636 @@
+                                       '/button[contains(@aria-label, "Save")]')
+
+        @property
+        def is_displayed(self):
+            return (self.table.is_displayed and
+                    (len(self.browser.elements(".//div[contains(@class,'spinner')]")) == 0))
+
+        def fill(self, values):
+            osp_security_group = values.get('osp_security_group')
+            osp_flavor = values.get('osp_flavor')
+            if osp_security_group or osp_flavor:
+                self.edit_instance_properties.click()
+                self.select_security_group.wait_displayed()
+                if osp_security_group:
+                    self.select_security_group.fill(osp_security_group)
+                if osp_flavor:
+                    self.select_flavor.fill(osp_flavor)
+                self.save_properties.click()
+            self.after_fill(was_change=True)
+
+        def after_fill(self, was_change):
+            self.parent.next_btn.click()
+
+    @View.nested
+    class advanced(View):  # noqa
+        pre_playbook = BootstrapSelect("preMigrationPlaybook")
+        post_playbook = BootstrapSelect("postMigrationPlaybook")
+        pre_checkbox = Text(locator='.//input[contains(@id, "pre_migration_select_all")]')
+        post_checkbox = Text(locator='.//input[contains(@id, "post_migration_select_all")]')
+
+        @property
+        def is_displayed(self):
+            return self.pre_playbook.is_displayed
+
+        def fill(self, values):
+            self.pre_playbook.wait_displayed("5s")
+            pre_playbook = values.get('pre_playbook')
+            post_playbook = values.get('post_playbook')
+            if pre_playbook:
+                self.pre_playbook.fill(pre_playbook)
+                self.pre_checkbox.click()
+            if post_playbook:
+                self.post_playbook.wait_displayed("5s")
+                self.post_playbook.fill(post_playbook)
+                self.post_checkbox.click()
+            self.after_fill(was_change=True)
+
+        def after_fill(self, was_change):
+            self.parent.next_btn.click()
+
+    @View.nested
+    class schedule(View):  # noqa
+        create = Button("Create")
+        run_migration = RadioGroup('.//div[contains(@id,"migration_plan_choice_radio")]')
+
+        @property
+        def is_displayed(self):
+            return self.run_migration.is_displayed
+
+        def fill(self, values):
+            if values:
+                self.run_migration.select("Start migration immediately")
+            self.after_fill(was_change=True)
+
+        def after_fill(self, was_change):
+            self.create.click()
+
+    @View.nested
+    class results(View):  # noqa
+        msg = Text('.//h3[contains(@id,"migration-plan-results-message")]')
+
+        @property
+        def is_displayed(self):
+            return self.msg.is_displayed
+
+    def after_fill(self, was_change):
```

Comment:
You may want to define these above the nested views, its not clear to which class this method and is_displayed belong.